Exercise_A/index.html

```diff
@@ -6,33 +6,217 @@
+            </div>
+          </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-thumbs-up fa-3x" style="color: #f05f40"></i>
+              <h3>Lots of effort</h3>
+              <p>We even changed some of the icons! We take the extra effort to make our designs truly original.</p>
+            </div>
+          </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-heart fa-3x" style="color: #f05f40"></i>
+              <h3>Made with Love</h3>
+              <p>Because nothing says hard work and talent like editing a few lines of text.</p>
+            </div>
+          </div>
+      
+          </div>
+      
+        </div>
+
+
+
+
+
+
+
+        <!-- Four pictures with overlay etc -->
+  <div class="container-fluid" id="fourIconsDiv">
+    <div class="row">
+
+      <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="fourIconsDiv">
+      <div class="adjustedPicture">
+        <img src="./assets/images/1.jpg">
+        <div class="overlay">
+          <div class="text"><span style="color: rgb(255, 251, 251, 0.8);">ONE THING </span><br> Look</div>
+        </div> 
+    </div>
+    </div>
+
+    <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="fourIconsDiv">
+      <div class="adjustedPicture">
+        <img src="./assets/images/2.jpg">
+        <div class="overlay">
+          <div class="text"><span style="color: rgb(255, 251, 251, 0.8);">TWO THING </span><br> At</div>
+        </div> 
+    </div>
+    </div>
+
+    <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="fourIconsDiv">
+      <div class="adjustedPicture">
+        <img src="./assets/images/3.jpg">
+        <div class="overlay">
+          <div class="text"><span style="color: rgb(255, 251, 251, 0.8);">THREE THING </span><br> This</div>
+        </div> 
+    </div>
+    </div>
+
+    <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="fourIconsDiv">
+      <div class="adjustedPicture">
+        <img src="./assets/images/4.jpg">
+        <div class="overlay">
+          <div class="text"><span style="color: rgb(255, 251, 251, 0.8);">FOUR THING </span><br> Bullshit</div>
+        </div> 
+    </div>
+    </div>
+
+    </div>
+  </div>
+
+
+  <!-- Grey div before contact me section-->
+
+  <div class="container-fluid" id="fourIconsDiv" style="background-color:rgb(58, 58, 58)">
+      <div class="d-flex justify-content-center align-content-center whiteText">
+      <h3>Seriously, it's this fucking template!</h3>
+      </div>
+      <div class="d-flex justify-content-center align-content-center">
+        <a href="#" class="btn btn-lg" style="background-color: white" href="#" role="button" id="learnMore">THAT'S AMAZING!</a>

```

Comment:
You could consider using more bootstrap utilities so that you do not need in-line styling and to reduce the size of the stylesheet.